css/example/Input.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Input</title>
+    <link rel="stylesheet" href="../components/Input.css">
+</head>
+<body>
+    <form class="input-form">
+        <div class="input-name">
+            <label for="username">Username</label>
+            <input class="input" type="text" name="username" required>
+        </div>
+
+        <div class="input-password">
+            <label for="password">Password</label>
+            <input class="input" type="password" name="password" required>
+        </div>
+
+        <div class="input-submit">
+            <a href="#"><input class="input" type="submit" name="submit"></a>
```

Comment:
I think there is no need of input tag inside hyperlink tag(a).

Reply:
removed (a) tag